weather-forecast/index.js

```diff
@@ -0,0 +1,33 @@
+const currentWeather = {
+  location: "Tbilisi",
+  temperature: 15,
+  weatherCondition: "snowy",
+  date: "12-04-2022",
+};
+
+function download() {
+  const l = document.getElementById("location");
+  const t = document.getElementById("temperature");
+  const w = document.getElementById("weatherCondition");
+  const d = document.getElementById("date");
+
+  l.lastChild.replaceWith(currentWeather.location);
```

Comment:
This code works, but it will be better to just reassign innerText property or current element (not to replace the lastChild), because semantically you are just changing the text

Reply:
fixed to innerText